src/Microsoft.OData.Core/UriParser/Binders/InBinder.cs

```diff
@@ -232,6 +232,13 @@ private static int ProcessDoubleQuotedStringItem(int start, string input, String
                 char next = input[k];
                 if (next == '"')
                 {
+                    // If prev and next are both double quotes, then it's an empty string.
+                    if (input[k - 1] == '"')
+                    {
+                        // We append \"\" so as to return "\"\"" instead of "".
+                        // This is to avoid passing an empty string to the ConstantNode.

```

Comment:
Curious to know, why can't we update the `ConstantNode` to make it allow empty strings?

Reply:
Oh I see, `\"\"` is how empty strings are represented in the `ConstantNode`?

Reply:
@habbes Yes, we use `""` instead of `String.Empty`